components/expenses/MarkExpenseAsPaidBtn.js

```diff
@@ -64,11 +64,15 @@ class MarkExpenseAsPaidBtn extends React.Component {
       unlock();
     } catch (e) {
       const error = getErrorFromGraphqlException(e).message;
-      this.setState({ error, loading: false });
+      this.setState({ error, loading: false }, this.passErrorMessage);
```

Comment:
Why do this in a `setState` callback? Do we still need to store the `error` in state since we don't display it anymore?

Reply:
again, you're right, that was unnecessary too in retrospect.